test/unittest/CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ foreach(NATIVE_TEST ${NATIVE_TESTS})
             ROOT_DIRECTORY ${LIBCYPHAL_ROOT}
             TARGET_EXECUTION_DEPENDS ${LOCAL_TEST_REPORT}
             OBJECT_LIBRARY ${LOCAL_TEST_LIB}
+            EXCLUDE_PATHS ${LIBCYPHAL_ROOT}external
```

Comment:
Are we missing an `/` here? I get it that `LIBCYPHAL_ROOT` contains a trailing `/` but it's a matter of readability